prospero-common/src/main/java/org/wildfly/prospero/api/RepositoryUtils.java

```diff
@@ -18,18 +18,21 @@
 package org.wildfly.prospero.api;
 
 import org.eclipse.aether.repository.RemoteRepository;
+import org.eclipse.aether.repository.RepositoryPolicy;
 import org.wildfly.channel.Repository;
 
-import static org.wildfly.prospero.api.ProvisioningDefinition.DEFAULT_REPOSITORY_POLICY;
-
 public class RepositoryUtils {
     public static Repository toChannelRepository(RemoteRepository r) {
         return new Repository(r.getId(), r.getUrl());
     }
 
     public static RemoteRepository toRemoteRepository(String id, String url) {
         return new RemoteRepository.Builder(id, "default", url)
-                .setPolicy(DEFAULT_REPOSITORY_POLICY)
+                .setPolicy(new RepositoryPolicy(true, RepositoryPolicy.UPDATE_POLICY_ALWAYS, RepositoryPolicy.CHECKSUM_POLICY_IGNORE))
```

Comment:
Why the `CHECKSUM_POLICY_IGNORE` policy? don't think we should ignore checksum errors

Reply:
Thanks for spotting, another forgotten bit. I was testing resolving manifests from local repo, that didn't have checksums...